resources/views/talks/show.blade.php

```diff
@@ -9,14 +9,21 @@
             <li><a href="/talks/{{ $talk->id }}">Talk: {{ $current->title }}</a></li>
         </ol>
 
+        @if ($talk->current()->id == $current->id)
```

Comment:
Is `$current` here the revision currently being displayed, while `$talk->current()` is the most recent revision? I think this could use some naming love.

Could probably also replace these checks with something like `@if ($revision->isCurrent())`